src/state/apis/zapper/validators.ts

```diff
@@ -660,603 +717,598 @@ const userNftTokenSchema = z.object({
+// parameters: [
+// {
+// name: 'addresses[]',
+// type: 'Query',
+// schema: z.array(z.string()),
+// },
+// ],
+// response: z.void(),
+// },
+// {
+// method: 'get',
+// path: '/v2/nft/balances/tokens',
+// requestFormat: 'json',
+// parameters: [
+// {
+// name: 'addresses[]',
+// type: 'Query',
+// schema: z.array(z.string()),
+// },
+// {
+// name: 'minEstimatedValueUsd',
+// type: 'Query',
+// schema: z.string().optional(),
+// },
+// {
+// name: 'search',
+// type: 'Query',
+// schema: z.string().optional(),
+// },
+// {
+// name: 'collectionAddresses[]',
+// type: 'Query',
+// schema: z.array(z.string()).optional(),
+// },
+// {
+// name: 'network',
+// type: 'Query',
+// schema: SupportedZapperNetworks.optional(),
+// },
+// {
+// name: 'limit',
+// type: 'Query',
+// schema: z.string().optional().default('25'),
+// },
+// {
+// name: 'cursor',
+// type: 'Query',
+// schema: z.string().optional(),
+// },
+// ],
+// response: z.void(),
+// },
+// {
+// method: 'get',
+// path: '/v2/nft/balances/tokens-totals',
+// requestFormat: 'json',
+// parameters: [
+// {
+// name: 'addresses[]',
+// type: 'Query',
+// schema: z.array(z.string()),
+// },
+// {
+// name: 'minEstimatedValueUsd',
+// type: 'Query',
+// schema: z.string().optional(),
+// },
+// {
+// name: 'search',
+// type: 'Query',
+// schema: z.string().optional(),
+// },
+// {
+// name: 'collectionAddresses[]',
+// type: 'Query',
+// schema: z.array(z.string()).optional(),
+// },
+// {
+// name: 'network',
+// type: 'Query',
+// schema: SupportedZapperNetworks.optional(),
+// },
+// ],
+// response: z.void(),
+// },
+// {
+// method: 'get',
+// path: '/v2/nft/collection/tokens',
+// requestFormat: 'json',
+// parameters: [
+// {
+// name: 'collectionAddress',
+// type: 'Query',
+// schema: z.string(),
+// },
+// {
+// name: 'network',
+// type: 'Query',
+// schema: SupportedZapperNetworks,
+// },
+// {
+// name: 'cursor',
+// type: 'Query',
+// schema: z.string().optional(),
+// },
+// {
+// name: 'tokenIds[]',
+// type: 'Query',
+// schema: z.array(z.string()).optional(),
+// },
+// ],
+// response: z.void(),
+// },
+// {
+// method: 'get',
+// path: '/v2/nft/user/tokens',
+// requestFormat: 'json',
+// alias: 'getV2NftUserTokens',
+// parameters: [
+// {
+// name: 'userAddress',
+// type: 'Query',
+// schema: z.string(),
+// },
+// {
+// name: 'network',
+// type: 'Query',
+// schema: SupportedZapperNetworks.optional(),
+// },
+// {
+// name: 'limit',
+// type: 'Query',
+// schema: z.string().optional().default('50'),
+// },
+// {
+// name: 'cursor',
+// type: 'Query',
+// schema: z.string().optional(),
+// },
+// ],
+// response: V2NftUserTokensResponse,
+// },
+// {
+// method: 'get',
+// path: '/v2/prices',
+// description: `Retrieve supported tokens and their prices`,
+// requestFormat: 'json',
+// parameters: [
+// {
+// name: 'network',
+// type: 'Query',
+// schema: SupportedZapperNetworks.optional(),
+// },
+// ],
+// response: z.void(),
+// },
+// ]
```

Comment:
cleanup

Reply:
All left here on purpose:
https://github.com/shapeshift/web/blob/deacaa016446abccfdeed84c903d9d8bd948ff6b/src/state/apis/zapper/validators.ts#L741

Happy to remove it if we feel like it's useless, but since comments have basically no runtime or build overhead, might as well

Reply:
Akschually yeah let's use git history if needs be - f5ea228561a58b97bd69e5d904b75e83fa3d9e98